go/chat/context.go

```diff
@@ -106,7 +102,8 @@ func CtxAddLogTags(ctx context.Context, env appTypeSource) context.Context {
 	rpcTags := make(map[string]interface{})
 	rpcTags["user-agent"] = libkb.UserAgent
 	rpcTags["platform"] = libkb.GetPlatformString()
-	rpcTags["apptype"] = env.GetAppType()
+	rpcTags["apptype"] = g.GetEnv().GetAppType()
+	rpcTags["slow-conn"] = g.GetEnv().GetSlowGregorConn()
```

Comment:
I'd rather not send this all the time, it is just wasted bandwidth on every call.

Reply:
fixed